### DISCUSSION TITLE
Speech input at runtime

### DISCUSSION THREAD

**Discussion by t-constantin** [microsoft/MixedRealityToolkit-Unity#9826] [Q&A]
Hi! Quick question, is it possible to add speech input keywords at runtime/dynamically to the speech commands?
As the list of SpeechCommands are read-only, is there a way to edit this list?

**Comment by david-c-kline** [comment]
@t-constantin, good question! let me consult with the team and have someone get back to you on this.

**Reply by polar-kev** [reply]
@RogPodge can you take a look at this? @t-constantin, we're currently doing a team hackathon so we should have a response for you next week.

**Reply by RogPodge** [reply]
Yeah I'll be able to take a look. My current intuition says that it's possible, but probably not super well supported/easy
